src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -205,6 +217,42 @@ public DriveTrain() {
     // robot.
     odometry =
         new DifferentialDriveOdometry(navx.getRotation2d(), getLeftDistance(), getRightDistance());
+
+    xSimStart.setNumber(2.0);
+    ySimStart.setNumber(2.0);
+    rotSimStart.setNumber(0.0);
+    readSimStart.setBoolean(false);
```

Comment:
Are these just for debug?

Reply:
fThey are default values to start with.  I just picked them randomly, but we can change in future if we find we are always setting up robot in certain place on the field?